client/js/recurring_donations/edit/index.es6

```diff
@@ -349,22 +348,22 @@ const updateAddressModal = state =>
   })
 
 
-const paydateForm = state =>
-  h('form', { on: {submit: state.submitPaydate$} }, [
-    h('p', 'Enter a day of the month (between 1 and 28) when you want to be charged for this donation.')
-  , h('p', 'This will fix your donations to that date each month for all future payments.')
-  , h('input.input--small', {
+const paydateForm = (state) =>
+  h('form', { on: { submit: state.submitPaydate$ } }, [
+    h('p', 'Enter a day of the month (between 1 and 28) when you want to be charged for this donation.'),
+    h('p', 'This will fix your donations to that date each month for all future payments.'),
+    h('input.input--small', {
       props: {
-        type: 'number'
-      , max: 28
-      , min: 1
-      , name: 'paydate'
-      , value: state.recDon$().paydate || 1
-      }
-    })
-  , h('br')
-  , button(R.pick(['loading$', 'error$'], state))
-  ])
+        type: 'number',
+        max: 28,
+        min: 1,
+        name: 'paydate',
+        value: state.recDon$().paydate || 1,
+      },
+    }),
+    h('br'),
+    button(pick(state, ['loading$', 'error$'])),
+  ]);
```

Comment:
Would it be possible to revert all of these changes except the `pick` change?

Reply:
yep!